styles/index.css

```diff
@@ -16,7 +16,7 @@
 /* End of Page Identifier Name */
 
 /* Button Styling */
-.add {
+#addButton {
     font-family: 'Rajdhani', sans-serif;
```

Comment:
I think we should just add this font to all descendents under body inside page.css? Or else we'll have to keep on declaring these in future.